ql/src/test/results/clientpositive/llap/vector_reuse_scratchcols.q.out

```diff
@@ -369,7 +369,7 @@ STAGE PLANS:
                     includeColumns: [0, 1, 2, 3, 4, 5, 6, 7, 9, 10, 11]
                     dataColumns: ctinyint:tinyint, csmallint:smallint, cint:int, cbigint:bigint, cfloat:float, cdouble:double, cstring1:string, cstring2:string, ctimestamp1:timestamp, ctimestamp2:timestamp, cboolean1:boolean, cboolean2:boolean
                     partitionColumnCount: 0
-                    scratchColumnTypeNames: [double, double, double, decimal(22,3), decimal(13,3), double, double, double, double, double, double, double, double]
+                    scratchColumnTypeNames: [double, double, double, decimal(22,3), decimal(13,3)/DECIMAL_64, double, double, double, double, double, double, double, double]
```

Comment:
any idea how this shows up as decimal(13,3)/DECIMAL_64 - would the decimal(13,3) part also be converted to decimal64?

Reply:
Yes, because we can safely store a decimal with (13,3) by scaling them up and storing in long vectors and tracking the scale. We do this until precision+scale is <=18.